.github/workflows/packing.yml

```
@@ -504,25 +513,55 @@ jobs:
         env:
           DEBIAN_FRONTEND: noninteractive
 
+      - name: Create venv
+        run: |
+          apt install -y python3-venv
+          python3 -m venv .venv
+
       - name: Install test requirements
         run: pip3 install -r requirements-test.txt
+        if: matrix.target.dist != 'bookworm' && matrix.target.dist != 'noble'
```

Please, add a comment with a reason of such conditions:
```suggestion
        if: matrix.target.dist != 'bookworm' && matrix.target.dist != 'noble'
```

---

Added.